### DISCUSSION TITLE
Error in Documentation Links

### DISCUSSION THREAD

**Discussion by KlekotOliwa** [DaveSkender/Stock.Indicators#1254] [Help and support]
Standard Deviation (volatility)

•  Z-Score

Both links currently lead to the materials on standard deviation. Additionally, the link for Z-Score is not functioning as expected and redirects to the standard deviation page.

Could someone please look into this and correct the links? It would be greatly appreciated.

Thank you for your attention to this matter.

Best regards,
Klekot

**Comment by DaveSkender** [comment]
Tracking in #1255, thank you :heart: for reporting!

**Comment by DaveSkender** [comment]
Just took a peek, both of these are found on the same page.  I can see how that might be confusing, but it's intentional.  See #1255 for more explanation.
